src/server/utils/authorization.js

```diff
@@ -104,6 +104,27 @@ export const requireWebsocketExchange = (exchange) => {
   }
 };
 
+export const getSegmentTraitsForUsers = async (userIds) => {
+  // TODO: one day we'll store this in something like redis
+  const r = getRethink();
+  const traits = await r.table('User')
+    .getAll(...userIds)
```

Comment:
did you test this with an array? I would think you'd have to write `.getAll(r.args(userIds))`